SetLargeAddressAware/LargeAddressAware.targets

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!--
+      Full path to the assembly that contains the MSBuild task
+    -->
+    <LargeAddressAwareAssemblyFile Condition=" '$(LargeAddressAwareAssemblyFile)' == '' ">$(MSBuildThisFileDirectory)net46\SetLargeAddressAware.dll</LargeAddressAwareAssemblyFile>
+  </PropertyGroup>
+
+  <!--
+    To run this target, set the property LargeAddressAware to true in your project.
+    <PropertyGroup>
+      <LargeAddressAware>true</LargeAddressAware>
+    </PropertyGroup>
+    
+    The target only runs after CoreCompile so it is incremental and does not need Inputs/Outputs
+  -->
+  <Target Name="SetLargeAddressAware"
+          AfterTargets="CoreCompile"
+          Condition=" '$(LargeAddressAware)' == 'true' And Exists('$(LargeAddressAwareAssemblyFile)') And '$(OutputType)' == 'Exe' And '$(PlatformTarget)' == 'x86' ">
```

Comment:
I think `OutputType=winexe` would be valid here as well.

Reply:
Oh yeah, good catch!

Reply:
Does MSBuild support parentheses? How would I change this expression?

`And ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'winexe') And `

would that work?

Reply:
Yeah, we support nested parens in logical expressions. It's basically the one part of MSBuild that has a real parser!

Reply:
lol I wouldn't call that a real parser :) I've seen some of that code :)

Reply:
How's this? https://github.com/KirillOsenkov/LargeAddressAware/commit/909a19781b5c70d1ab9f9e0cb467a28e60579504

Reply:
👍